src/api/Readme.md

```diff
@@ -0,0 +1,18 @@
+
+# How-To Generate OpenAPI Sepecification
+
+
+## Application API
+
+- Get the OpenAPI specification from the [prod swagger page of SelfService](https://apissappssprodwe01.azurewebsites.net/swagger/v1/swagger.json).
```

Comment:
I would only mention the APIM export variant, since this is the preferred way imho.

In the future the API im APIM might look differnt to the backend APIs.